### DISCUSSION TITLE
Any chance Wirt/Adria could sell Staves?

### DISCUSSION THREAD

**Discussion by FishGaming** [diasurgical/DevilutionX#6320] [Q&A] [answered]
I'm not sure which most people would want to have, but since I like being able to have the chance to get any item that is in the game, could there be any consideration on making it where one of the NPC's above could sell staves so we could have a chance at the qlvl 60's on staves?
Of course might not be the biggest problem anyway since that would only give hydra from what I could see. That's why I suggested both NPC's. Of course Adria would produce it easier than Wirt, but if staves were added to Wirt it could really increase the random pool which could not only make it much harder to get the item you want from Wirt, but it could also sometimes make it easier if helped the failing of meeting proper criteria the first 250 attempts Wirt goes through when generating you the item.

**Comment by qndel** [comment, marked as answer]
No

**Reply by julealgon** [reply]
LOL @qndel , your bluntness is something else 😆 

I think selling staves on Wirt could be interesting... would be a seamless way of allowing some of the impossible affixes today, and would make sense not only for Sorcerer but for Monk as well (they really messed this up in Hellfire).

Didn't they add non-skill staves to Griswold in Hellfire BTW? Extending that to Wirt makes even more sense if so.

**Reply by DakkJaniels** [reply]
Why should this be done in DevilutionX and not a mod? Is this a bug?

**Reply by StephenCWills** [reply]
> Didn't they add non-skill staves to Griswold in Hellfire BTW? Extending that to Wirt makes even more sense if so.

Wirt already sells staves in Hellfire.

**Comment by Trihedraf** [comment]
DevilutionX does not change game mechanics. That would be a need to done in a separate mod.

**Comment by FishGaming** [comment]
Although this is technically a game mechanic change, it is a would of been a fix that the devs never took care of, but it appears that Wirt already sells staves in Hellfire so the feature is already there. Thanks for the answers guys!

**Comment by qndel** [comment]
> Although this is technically a game mechanic change, it is a would of been a fix that the devs never took care of, but it appears that Wirt already sells staves in Hellfire so the feature is already there. Thanks for the answers guys!

https://www.grammarly.com/blog/would-of-could-of-should-of/
